spec/fastsync/vdd/fastsyncL2.tla

```diff
@@ -337,7 +337,7 @@ FindPeerToServe(bPool, h) ==
     \* pick a peer that can serve request for height h that has minimum number of pending requests
     ELSE CHOOSE p \in peersThatCanServe: \A q \in peersThatCanServe:       
             /\ NumOfPendingRequests(bPool, p) <= NumOfPendingRequests(bPool, q) 
-            /\ p <= q
+            \*/\ p <= q \* otherwise, TLC reports an error in CHOOSE
```

Comment:
I have commented out `p <= q`, as TLC complained about no element satisfying the predicate. I do not understand why.

Reply:
Removed.